extensions/git/src/commands.ts

```diff
@@ -38,6 +38,25 @@ class CheckoutItem implements QuickPickItem {
 	}
 }
 
+class TagItem implements QuickPickItem {
+
+	get label(): string { return (this.tag.name || '').substr(0, 20); }
```

Comment:
Hi @joaomoreno , I would like your input on the label length allowed for the quick pick items. For now, I am arbitrarily cropping the tag label to 20 characters , this rest of the line is taken by the tag message. I am wondering if 20 characters is ok?

Reply:
Sure.